manifests/t/TexasInstruments/TI-Nspire/CXPremiumTeacher/5.3.2.129/TexasInstruments.TI-Nspire.CXPremiumTeacher.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TexasInstruments.TI-Nspire.CXPremiumTeacher
+PackageVersion: 5.3.2.129
+PackageLocale: en-US
+Publisher: Texas Instruments
+PublisherUrl: https://education.ti.com
+PublisherSupportUrl: https://education.ti.com/en/customer-support
+PrivacyUrl: https://www.ti.com/legal/terms-conditions/privacy-policy.html
+Author: Texas Instruments Inc.
+PackageName: TI-Nspire CX Premium Teacher

```

```suggestion
PackageName: TI-Nspire CX Premium Teacher Software
```
This needs to match Control Panel.